Libplanet.Tests/Action/AccountStateDeltaImplTest.cs

```diff
@@ -34,20 +35,20 @@ public void CreateNullDelta()
         {
             IAccountStateDelta delta = new AccountStateDeltaImpl(GetState);
             Assert.Empty(delta.UpdatedAddresses);
-            Assert.Equal("a", delta.GetState(_addr[0]));
-            Assert.Equal("b", delta.GetState(_addr[1]));
+            Assert.Equal("a", (Text)delta.GetState(_addr[0]));
+            Assert.Equal("b", (Text)delta.GetState(_addr[1]));
             Assert.Null(delta.GetState(_addr[2]));
         }
 
         [Fact]
         public void GetSetState()
         {
             IAccountStateDelta init = new AccountStateDeltaImpl(GetState);
-            IAccountStateDelta a = init.SetState(_addr[0], "A");
-            Assert.Equal("A", a.GetState(_addr[0]));
-            Assert.Equal("a", init.GetState(_addr[0]));
-            Assert.Equal("b", a.GetState(_addr[1]));
-            Assert.Equal("b", init.GetState(_addr[1]));
+            IAccountStateDelta a = init.SetState(_addr[0], "A".ToBencodex());
```

Comment:
There seem two ways to convert a `string` object to a `Text` object: casting operator and `.ToBencodex()` method, and in this code two ways coexist.  Is there any guideline to choose the appropriate way for each case between two ways?

Personally, `.ToBencodex()` has a shortcoming that it hides the resulting type; although we readers could find it's something `IValue`, it's difficult to guess its concrete type (e.g., `Text`, not just `IValue`).

Reply:
1. You're right. I'll prepare it.
2. Yes, so I tried to rename the names like `AsText`, `AsInteger` and `AsBinary` etc..

Reply:
In my thought, `v.AsText()` looks better than `(Text)v`.